server/resp/src/main/java/org/infinispan/server/resp/commands/connection/PING.java

```diff
@@ -30,6 +30,8 @@ public CompletionStage<RespRequestHandler> perform(Resp3Handler handler, Channel
       if (arguments.isEmpty()) {
          ByteBufferUtils.stringToByteBufAscii("$4\r\nPONG\r\n", handler.allocator());
          return handler.myStage();
+      } else if (arguments.size()==1) {
```

Comment:
Looking at the command page, it seems it can have more arguments. `PING hello world`.

Reply:
if you mean this https://redis.io/docs/latest/commands/ping/
the example has 1 argument: `PING "hello world"`
I tried `PING hello world` on redis:
```
127.0.0.1:6379> ping hello world
(error) ERR wrong number of arguments for 'ping' command
```